src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (num) => {
+      return (calculator.result += num);
+    },
+
+    subtract: (num) => {
+      return (calculator.result -= num);
+    },
+
+    multiply: (num) => {
+      return (calculator.result *= num);
+    },
+
+    divide: function (num) {
+      return (calculator.result /= num);
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics